src/components/gameScreen/gameScreen.container.js

```diff
@@ -37,19 +46,47 @@ export const GameScreenContainer = () => {
     return () => clearTimeout(timeout)
   }, [counter]);
 
-  useEffect(() => {
-    if (questionNumber  === numberOfQuestions ) {
-      navigateToGameOver();
+  const timerColors = {
+    inActiveStrokeColor: '#b2b2d7',
+    activeStrokeColor: '#d9b1ff',
+    circleBackgroundColor: '#fff',
+  }
```

Comment:
don't write constants like this inside of components, because they will be recreated each time. Write this const outside of component